ipyleaflet/leaflet.py

```diff
@@ -1841,11 +1841,11 @@ def legends(self, legends):
         self.legend = legends
 
     @property
-    def positionning(self):
+    def positioning(self):
```

Comment:
Maybe we could keep the old `positionning` and make it call the new `positioning`, for compatibility reasons?

Reply:
Just added `positionning`.